cli_tools/common/utils/logging/firelog/log_entry.go

```diff
@@ -0,0 +1,175 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package firelog
```

Comment:
If the data transport team ever wrote a go client, and you wanted to adopt it in this tool, would this conflict?

Reply:
No, it's bounded to the project name + path as well. But as discussed, I would remove this word as it's internal